module/delete.py

```diff
@@ -1,25 +1,29 @@
-from module.get_on_sale import get_on_sale
 from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
 from telegram.ext import CallbackContext
+from module.create_connection import create_connection
+from module.shared import DB_PATH
 
 def delete(update: Update, context: CallbackContext) -> None:
     chat_id= update.effective_chat.id
     message=update.message.text
     if message =='/elimina':
         user ="@"+str(context.bot.get_chat(chat_id)['username'])
-        df = get_on_sale()
-        condition = df['Venditore'] == user
-        ret = df[condition].index.tolist()
-
-        if len(ret)!=0:
-            context.bot.send_message(chat_id, "Quale libro vuoi eliminare?\n")
-            res=""
-            for i in range(len(ret)):
-                res+='n°: ' + str(i+1) + '\n' + 'ISBN: ' +str(df.iloc[ret[i]][0]) + '\n' + 'Titolo: ' +str(df.iloc[ret[i]][1]) + '\n' + 'Autori: '+str(df.iloc[ret[i]][2]) + '\n' + 'Venditore: ' + str(df.iloc[ret[i]][3]) + '\n' + 'Prezzo: ' + str(df.iloc[ret[i]][4]) + ' €\n\n' 
-            keyboard = [ [InlineKeyboardButton(str(i+1), callback_data=(ret[i]))] for i in range(len(ret)) ]
-            reply_markup = InlineKeyboardMarkup(keyboard)
-            update.message.reply_text(res, reply_markup=reply_markup)
+        conn = create_connection(DB_PATH)
+        if conn:
+            cur = conn.cursor()
+            cur.execute("SELECT rowid, * FROM Market WHERE Venditore=?", (user,))
+            rows = cur.fetchall()
+            if len(rows):
```

Comment:
I like this kind of style, but, as pylint wanrs, you have to put `len(rows) > 0` or `if rows:`
```suggestion
            if len(rows) > 0:
```

sources:
https://stackoverflow.com/a/43476778/5081063
https://stackoverflow.com/a/61598621/5081063